src/app/components/cds/buttons-checkboxes-chips/ButtonMain.module.css

```diff
@@ -28,6 +28,7 @@
     display: flex;
     /* stylelint-disable-next-line scale-unlimited/declaration-strict-value */
     font-size: 20px;
+    width: fit-content;
```

Comment:
@brianfleming I don't *think* this will affect anything currently in production, and the Sandbox looks good still. Basically this is just to prevent the width of the button icon from inheriting from a parent

Reply:
Makes sense...I will click around and see if anything weird happens, but I highly doubt it